Fixed #26029 -- Allowed configuring custom file storage backends. (#16442)
django/django: docs/howto/custom-file-storage.txt

If a file with ``name`` already exists, ``get_alternative_name()`` is called to
obtain an alternative name.

.. _using-custom-storage-engine:

Use your custom storage engine
==============================

.. versionadded:: 4.2

The first step to using your custom storage with Django is to tell Django about
the file storage backend you'll be using. This is done using the
:setting:`STORAGES` setting. This setting maps storage aliases, which are a way
to refer to a specific storage throughout Django, to a dictionary of settings
for that specific storage backend. The settings in the inner dictionaries are
described fully in the :setting:`STORAGES` documentation.

> ``example`` storage can be retrieved from the
> ``django.core.files.storage.storages`` dictionary::

Reviewer: ```suggestion
Storages are then accessed by alias from from the
``django.core.files.storage.storages`` dictionary::
```

    from django.core.files.storage import storages

>     example_storage = storages['example']

Reviewer: ```suggestion
    example_storage = storages["example"]
```

?
